CHANGELOG.md

```diff
@@ -0,0 +1,256 @@
+# Change Log for AWS IoT Device SDK for Embedded C
+
+## 202009.00 (September 2020)
+
+This release is **NOT** backwards compatible with the v3 version of AWS IoT Device SDK for Embedded C.
```

Comment:
hmm. I am not an expert on docs, but can we start section about 202009.00 with what it is, rather than what it is not?

Reply:
That's a good point. Would it be fine if I move this sentence toward the end of the 202009.00 section?

Reply:
Looks good

Reply:
Should we speak to the new versioning scheme we are introducing here ? Perhaps just point to a description of it that exists here - https://docs.aws.amazon.com/freertos/latest/userguide/what-is-freertos.html

Reply:
Anything to say about CBMC which I believe is an additional value add from last release. Please disregard if this has already been discussed.

Reply:
Something like this:
"This release cuts over to a date-based versioning scheme. More information about the versioning scheme can be referred in the [FreeRTOS AWS docs landing page](https://docs.aws.amazon.com/freertos/latest/userguide/what-is-freertos.html)

Reply:
that link may not be the right link. Tanmoy is working on a post that will explain the new versioning and CBMC. I think we should link it from the changelog. Pls check with Tanmoy.

Reply:
Agree with Amit. Another thing to consider is speak about the yaml file if it is being introduced with this release.

Reply:
To prevent further delay on this PR, I will make the changes of this discussion thread in a follow-up PR.